jobs/build.Jenkinsfile

```diff
@@ -347,6 +347,17 @@ lock(resource: "build-${params.STREAM}") {
         // Build the remaining artifacts
         stage("Build Artifacts") {
             pipeutils.build_artifacts(pipecfg, params.STREAM, basearch)
+
+            // Stop the build if the kernel + kernel-rt versions do not match
```

Comment:
A good bit of context to add to this comment is that this should only apply to x86_64 builds (IIUC).

Reply:
That's correct. I thought about adding that, but it's in the x86_64 file, so I opted to not add the information. I'll update though comment though to be clear about what's going on.

Reply:
It'll also help if we decide to change the way we build x86_64 using `build.jenkinsfile`

Reply:
updated